src/test/java/it/pagopa/ecommerce/commons/TransactionTestUtils.java

```diff
@@ -48,20 +55,35 @@ public static TransactionActivationRequested transactionActivationRequested(Stri
     public static TransactionActivatedEvent transactionActivateEvent() {
         return new TransactionActivatedEvent(
                 TRANSACTION_ID,
-                RPT_ID,
-                PAYMENT_TOKEN,
-                new TransactionActivatedData(DESCRIPTION, AMOUNT, EMAIL, FAULT_CODE, FAULT_CODE_STRING, PAYMENT_TOKEN)
+                Arrays.asList(new it.pagopa.ecommerce.commons.documents.NoticeCode(PAYMENT_TOKEN, RPT_ID, null, null)),
```

Comment:
```suggestion
                Arrays.asList(new it.pagopa.ecommerce.commons.documents.NoticeCode(PAYMENT_TOKEN, RPT_ID, AMOUNT, NOTICE_DESCRIPTION)),
```

(`NOTICE_DESCRIPTION` is not present yet)